src/commands/schemaUpload.ts

```diff
@@ -27,16 +27,15 @@ const logger = new Logger("commands.schemaUpload");
  */
 
 /**
- * Wrapper around {@linkcode uploadSchemaFromFile}, triggered from an inline action on a schema
- * subject container in the Schemas view.
+ * Wrapper around {@linkcode uploadSchemaFromFile}, triggered from an inline actions:
+ *  1. On a Subect treeitem in the Schemas view (passing in a Subject)
+ *  2. On one of a topic's schema subject group in the Topics view (passing in a ContainerTreeItem<Schema>)
```

Comment:
Is this supposed to be a `Subject` and a `Schema`? Or a `Subject` and a `SubjectTreeItem`?

Reply:
I've read through more, and I think I see where I was confused -- this basically splits it into "was this called from the Topics view (ContainerTreeItem<Schema>) or Schemas view (Subject)", right?

Reply:
Respelled since is now much simpler, only ever called with a Subject.